index.html

```diff
@@ -46,9 +46,10 @@ <h1>Skiafy</h1>
         <br/>
         <span class="limitation">Works only with fill attributes & expects colors in form #FFF or #FFFFFF.</span>
       </label>
+      <label><input id="fix-viewbox" type="checkbox"> Translate and scale the viewbox to match the width of the svg</label><br/>
```

Comment:
Should I enable this checkbox by default? We usually don't get the right result if the width/height of the svg doesn't match the viewbox. If it matches, this checkbox is a no-op

Reply:
Only thing I'm worried about is how this breaks in cases we haven't considered like a non-square size, or unspecified size, or non-square viewbox. Therefore it might be safer to not check it by default.

Also can we update the text to

"""
Normalize viewbox and path coordinates to match the specified width of the SVG (assuming square dimensions).
"""